src/Tizen.NUI/src/public/WebView/WebSettings.cs

```diff
@@ -78,7 +78,7 @@ public bool SpatialNavigationEnabled
         /// <summary>
         /// Default Font Size.
```

Comment:
Default font size

Reply:
Done.
I updated it in another patch https://github.com/Samsung/TizenFX/pull/3520